iroha/src/lib.rs

```diff
@@ -311,7 +312,7 @@ const GENESIS_PATH: &str = "genesis.json";
 pub struct Arguments {
     /// Set this flag on the peer that should submit genesis on the network initial start.
     #[cfg_attr(feature = "cli", structopt(long))]
-    pub submit_genesis: bool,
+    pub submit_genesis: u8,
```

Comment:
Why did it became `u8`?

Reply:
It should not, but CLI was complaining about bool parameter, that's why I made it u8. Could you give me a better option?